client/pages/pitch.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { PitchDetector } from 'pitchy';
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+import { abs } from 'mathjs';
```

Comment:
The `Math` object already has this. Please `npm uninstall mathjs`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math#static_methods